wzl/src/components/types.ts

```diff
@@ -1,11 +1,49 @@
-import { ExtractPropTypes } from "vue";
+import { ComputedRef, ExtractPropTypes, InjectionKey, PropType } from "vue";
 
-//  定义 Props
+/** 列配置类型 */
+export interface TableColumn {
+  key: string | number;
+  align?: "start" | "end";
+  width?: string | number;
+  title?: string;
+  sortable?: TableColumnSortable;
+}
+
+export interface TableColumnSortable<T = any> {
+  orderBy?: "ascend" | "descend";
+  sorter?: (curr: T, next: T) => number;
+}
+
+// export interface TableData {
+//     key: string | number | symbol;
+//     [key: string]: any;
+// }
+
+/** 传进 WuTable 的 props 都在此定义 */
 export const tableProps = {
-  test: {
-    type: Boolean,
-    default: false,
+  dataSource: {
+    type: Array as PropType<any[]>,
```

Comment:
```suggestion
    type: Array as PropType<Record<string, any>[]>,,
```
这里建议不要将 Vue 的属性混入到这里，将他挪动到对应的文件夹，这里应该只有 TableProps 这样的属性就可以了。

Reply:
fixed